example_compensation/mix.lock

```diff
@@ -1,3 +1,8 @@
+# SPDX-FileCopyrightText: 2023 Digit
```

Comment:
Could you add `example_compensation/mix.lock` to the `REUSE.toml` so avoid the copyright being added to it? (Add to CC0 section)